angular/.editorconfig

```diff
@@ -0,0 +1,16 @@
+# Editor configuration, see https://editorconfig.org
```

Comment:
Maybe a good file to have the repo root?

Reply:
Moved it to root